scripts/mdbook-forc-documenter/Cargo.toml

```diff
@@ -20,3 +20,4 @@ mdbook = { version = "0.4", default-features = false }
 semver = "1.0"
 serde = "1.0"
 serde_json = "1.0"
+tracing = "0.1"
```

Comment:
Will `tracing`'s `error!` macros ever get output by `mdbook build`? `mdbook` would have to setup a subscriber internally right? Does it do this?

If not, perhaps it's worth just sticking to `eprintln` seeing as this is just useful for the mdbook preprocessor that we use internally.

Reply:
Oh, I missed that one, my bad. I am taking it back to `eprintln` and `println`